docs/content/api/workflows.md

```diff
@@ -0,0 +1,220 @@
+---
+title: TRISA Workflows
+date: 2024-05-27T08:32:06-05:00
+lastmod: 2024-05-27T08:32:06-05:00
+description: "Various workflow scenarios for handling travel rule information exchanges"
+weight: 10
+---
+
+In TRISA v1.1 an optional `transfer_state` field was added to the secure envelope to assist TRISA implementers managing various TRISA data exchange workflows. While completely optional, the `transfer_state` signals to the counterparty (the recipient of the secure envelope) the intent of the current transfer message. The transfer state is an enum, and the available states are described below:
+
+| Code | State | Description |
+|---|---|---|
+| 0 | UNSPECIFIED | The transfer state is unknown or purposefully not specified (default case) |
+| 1 | STARTED | This is the first message in the TRISA workflow |
+| 2 | PENDING | Action is required by the sending party, await a following RPC |
+| 3 | REVIEW | Action is required by the receiving party (rarely used in transfers, useful for applications) |
+| 4 | REPAIR | Some part of the payload of the TRISA exchange requires repair (attached to error envelopes) |
+| 5 | ACCEPTED | The TRISA exchange is accepted and the counterparty is awaiting the on-chain transaction |
+| 6 | COMPLETED | The TRISA exchange and the on-chain transaction have been completed |
+| 7 | REJECTED | The TRISA exchange is rejected and no on-chain transaction should proceed (attached to error envelopes) |
+
+The state transitions are described by the following flow-chart:
+
+```mermaid
+flowchart LR
+    STARTED
+    autoAccept{{Review Required?}}
+    STARTED-->autoAccept
+    autoAccept -- Yes --> PENDING
+    autoAccept -- No --> ACCEPTED
+    accept{{Accept Transfer?}}
+    reject{{Retry and Fix?}}
+    PENDING-->accept
+    accept -- Yes --> ACCEPTED
+    accept -- No --> reject
```

Comment:
Should `reject` be lowercase here? Just checking for consistency.

Reply:
Yes, sorry for the confusion - it's referencing the `reject` decision defined on Line 32 -- in the diagram it displays as "Retry and Fix?" -- it's a variable definition that is different than the "REJECTED" node.